src/server/array_func.c

```diff
@@ -1077,13 +1042,13 @@ mk_subjob_id(job *parent, int offset)
  * 		cvt-range - convert entries in subjob index table which are in "state"
  * 		to a range of indices of subjobs.  range will be of form:
  * 		X,X-Y:Z,...
- * @param[in]	t - pointer to subjob index table
+ * @param[in]	pjob - job pointer
  * @param[in]	state -  job state.
  * @return	Pointer to static buffer
  * @par	MT-safe: No - uses a global buffer, "buf" and "buflen".
  */
 char *
-cvt_range(struct ajtrkhd *t, int state)
+cvt_range(job *pjob, int state)
 {
 	unsigned int f;	/* first of a pair or range   */
```

Comment:
Again, I dont like the choice of variable names here. Can we have something meaningful?